src/Grid/grid.jl

```diff
@@ -709,6 +709,18 @@ end
 @inline getcoordinates(grid::AbstractGrid, cell::CellIndex) = getcoordinates(grid, cell.idx)
 @inline getcoordinates(grid::AbstractGrid, face::FaceIndex) = getcoordinates(grid, face.idx[1])
 
+function cellnodes!(global_nodes::Vector{Int}, grid::AbstractGrid, i::Int)
+    cell = getcells(grid, i)
+    cellnodes!(global_nodes, grid, cell)
+end
+function cellnodes!(global_nodes::Vector{Int}, grid::AbstractGrid, cell::AbstractCell)
```

Comment:
Where is this method used? It didn't exist before.

Reply:
In the function just above, it is a function barrier around the `cell` object. This has an impact for mixed grids, though mostly when `grid.cells` is poorly typed (i.e. `Vector{AbstractCell}`).

Reply:
Okay, perhaps make it internal and add a comment then?

Reply:
Isn't that only something you would use internally then though? I see it can be used e.g. here: https://github.com/Ferrite-FEM/Ferrite.jl/blob/9de9677fb3fb0e78601271a39c123b04ee2636f6/src/iterators.jl#L87-L94 but that is internal use.

Also no need to pass the grid in this case.

Reply:
I think it would be ok to have it external (if the grid input is removed) since not everyone is using the iterator interfaces. To avoid ambiguities for e.g. `cellnodes!(global_nodeids::AbstractVector, cell::MyCell)` , perhaps 
```suggestion
cellnodes!(global_nodes::AbstractVector, cell::AbstractCell) = _cellnodes!(global_coords, cell)
function _cellnodes!(global_nodes::Vector{Int}, cell::AbstractCell)
```